### PR TITLE
Add a check on hide-keyboard.directive.ts

### PR DESCRIPTION
**Pull request by roshantandukar**
Can we add this check please?

### CODE REVIEW

File: projects/hide-keyboard/src/lib/hide-keyboard.directive.ts [hunk 1]
```diff
@@ -45,7 +45,9 @@ export class HideKeyboardDirective {
 		this.el.nativeElement.readOnly = value;
 		if (this.el.nativeElement.children && this.el.nativeElement.children.length > 0) {
 			this.el.nativeElement.children[0].readOnly = value;
-			this.el.nativeElement.children[1].readOnly = value;
+      if(this.el.nativeElement.children[1] !== undefined) {
```
Comment (roshantandukar) [inline projects/hide-keyboard/src/lib/hide-keyboard.directive.ts:48]: We can also use this check if you would like:
`if (this.el.nativeElement.children.length > 1)`

**Review by LFQuintero — APPROVED** [review]

**Comment by roshantandukar** [comment]
Creating a new PR with length check

**Comment by heyligengregory** [comment]
Approved & new version published

**Comment by roshantandukar** [comment]
Thanks a lot. 

Roshan..

**Pull request closed without merging**